lang/src/main/kotlin/org/partiql/lang/compiler/PartiQLCompilerBuilder.kt

```diff
@@ -84,12 +87,25 @@ class PartiQLCompilerBuilder private constructor() {
             JoinRelationalOperatorFactoryDefault,
             OffsetRelationalOperatorFactoryDefault,
             LimitRelationalOperatorFactoryDefault,
-            LetRelationalOperatorFactoryDefault,
+            LetRelationalOperatorFactoryDefault
+        )
+
+        /**
+         * A collection of all the default relational operator implementations provided by PartiQL that are in the experimental stage.
+         * This is necessary so users do not need the experimental annotation is they want to use the stabilized operators only. (after PartiQLCompilerPipeline graduates from experimental)
+         */
+        @ExperimentalWindowFunctions
+        private val EXPERIMENTAL_RELATIONAL_OPERATOR_FACTORIES = listOf(
             WindowRelationalOperatorFactoryDefault
         )
 
         @JvmStatic
         fun standard() = PartiQLCompilerBuilder()
+
+        @ExperimentalWindowFunctions
+        @JvmStatic
+        // This is a hack to inject the experimental operators to the compiler.
+        fun experimental() = PartiQLCompilerBuilder().customOperatorFactories(EXPERIMENTAL_RELATIONAL_OPERATOR_FACTORIES)
```

Comment:
As we discussed in-person, this method would be addressing some sort of a possible need of a PartQL-*language* customer who might be worried about running code containing features we consider experimental, even if the customer does not use these features. 

This need is different from what @OptIn is for, which is for communicating to a PartiQL-*library* customer that certain of our APIs are not final yet. 

So, let's go without `experimental()` for now.
Instead, WindowRelationalOperatorFactoryDefault should just be added to DEFAULT_RELATIONAL_OPERATOR_FACTORIES, and get the @OptIn(@ExperimentalWindowFunctions::class) annotation. 
(The latter point is a follow up to what I said before. @OptIn will be more neat on WindowRelationalOperatorFactoryDefault than on DEFAULT_RELATIONAL_OPERATOR_FACTORIES)